protos/telemetry_server/telemetry_server.proto

```diff
@@ -131,6 +129,73 @@ message PublishUnixEpochTimeRequest {
     uint64 time_us = 1; // The next 'unix epoch time' status
 }
 
+// Publish Responses
```

Comment:
```suggestion
```

Request and Response messages don't need documentation (it's not parsed, the rest of the documentation is actually used by the generated code). See [here](https://github.com/mavlink/MAVSDK-Proto/blob/main/protos/telemetry/telemetry.proto#L150-L153) for instance, there is no documentation around it :+1: